### DISCUSSION TITLE
Loss of capacity due to Li plating in Modelling coupled degradation mechanisms in PyBaMM example

### DISCUSSION THREAD

**Discussion by Akila1993** [pybamm-team/PyBaMM#3747] [Q&A]
Hello Everyone @brosaplanella @tinosulzer ,

I have tried to obtain the Loss of capacity due to the Li plating model in [Modelling coupled degradation mechanisms in PyBaMM](https://docs.pybamm.org/en/latest/source/examples/notebooks/models/coupled-degradation.html) in this example by @DrSOKane. 

The resulting plot is as below, please note that only Li plating options are enabled. Could someone please explain this behavior to me? Why it is not showing a continuous capacity loss and it follows the charge-discharge cycle pattern? Is it due to reversible nature of Li plating.

<img width="344" alt="image" src="https://github.com/pybamm-team/PyBaMM/assets/125870976/b29f1917-d8ed-4e09-93b8-0492f04f39b7">

**Comment by DrSOKane** [comment]
It is due to the reversible nature of plating, correct.
